docker/databricks/Dockerfile

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+############# 
+# Combine all below Dockerfiles together:
+# https://github.com/databricks/containers/blob/master/ubuntu/gpu/cuda-11/base/Dockerfile
+# https://github.com/databricks/containers/blob/master/ubuntu/minimal/Dockerfile
+# https://github.com/databricks/containers/blob/master/ubuntu/python/Dockerfile
+# https://github.com/databricks/containers/blob/master/ubuntu/dbfsfuse/Dockerfile
+# https://github.com/databricks/containers/blob/master/ubuntu/standard/Dockerfile
+# https://github.com/databricks/containers/blob/master/experimental/ubuntu/ganglia/Dockerfile
+#############
+ARG CUDA_VERSION=11.3.1
```

Comment:
we want the cuda version to be great then 11.5 so we can use the async memory allocator

Reply:
updated to 11.5.2